Polyhedron/include/CGAL/boost/graph/properties_Polyhedron_3.h

```diff
@@ -30,13 +30,61 @@
 #include <CGAL/number_utils.h>
 #include <boost/shared_ptr.hpp>
 #include <CGAL/boost/graph/internal/Has_member_id.h>
+#include <boost/unordered_map.hpp>
 
 #define CGAL_HDS_PARAM_ template < class Traits, class Items, class Alloc> class HDS
 
 namespace CGAL {
 
 namespace internal {
 
+template <typename Poly, typename K, typename V>
+struct Dynamic_polyhedron_property_map {
+
```

Comment:
There is no reason this should be specific to Polyhedron. IMO, this should be the default implementation. That way the property mecanism will work out of the box for any model of the FaceGraph concept (instead of having to define some specialization and overloads so that it is working).